examples/elasticsearch-operator/elasticsearch-enterprise-search.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: elasticsearch-enterprise-search-sample
+spec:
+  components:
+    - type: elasticsearch-enterprise-search
+      name: elasticsearch-enterprise-search
+      properties:
+        version: 8.6.2
+        count: 1
+        elasticsearchRef:
+          name: elasticsearch-cluster
```

Comment:
how it works?

Reply:
@wonderflow This YAML is just to run Web GUI of elasticsearch-enterprise, For the actual implementation of elasticsearch users will have to follow the docs.